tox.ini

```diff
@@ -1,55 +1,56 @@
 [tox]
-envlist = py38-django{22,30,31}-celery{44,50}-drf{39,310,311,312},quality,docs
+envlist = py38-django{22,30,31,32}-celery{44,50}-drf{311,312},quality,docs
```

Comment:
In platform `djangorestframework==3.12.4` version is on prod, we can safely drop earlier version and add new one.
Try some thing like this `-drf{312,latest}`